frontend/packages/ceph-storage-plugin/src/components/kms-config/ibm-kms-config.tsx

```diff
@@ -1,7 +0,0 @@
-import * as React from 'react';
-import { KMSConfigureProps } from './providers';
-import './kms-config.scss';
-
-export const IbmKmsConfigure: React.FC<KMSConfigureProps> = () => {
-  return <></>;
-};
```

Comment:
added this file in previous commit as a dummy... have now removed it after creating `frontend/packages/ceph-storage-plugin/src/components/kms-config/hpcs-config.tsx` for the HPCS component.